src/client/game.cpp

```diff
@@ -2938,8 +2938,14 @@ void Game::updateChat(f32 dtime, const v2u32 &screensize)
 	chat_backend->step(dtime);
```

Comment:
Also if you don't mind you can remove the `screensize` argument to this function, it's unused.